### DISCUSSION TITLE
새로 데이터팀을 빌딩하고자 합니다.

### DISCUSSION THREAD

**Discussion by jsaang** [playinpap/awesome-data-and-analytics-governance#9] [Q&A]
새로 데이터팀을 빌딩하고자 할 때, 어떤 직군의 분들을 모셔야 하며 이분들의 R&R을 어떻게 정의하면 좋을까요?

**Comment by not-for-me** [comment]
안녕하세요
저는 현재 조직에서 데이터팀을 새로 빌딩 중에 있고,
이전 조직에서도 약 3년간 처음부터 데이터팀을 빌딩을 했었습니다.
주신 질문에 대하여 여러 의견 중 하나로 봐주시고 제 작은 경험을 나눠보고 싶습니다.

---
개인적인 결론부터 말씀드리면.
> 데이터 팀 빌딩 시 필요한 멤버는 전적으로 현재 조직이 풀어야 하는 당면한 문제에 맞춰서 셋업해야 한다고 생각합니다.

두 번의 경험이 각기 케이스가 달라서 두 가지 사례를 일반화하여 어떻게 진행했는지를 공유하겠습니다.

## 사례1: 큰 규모 조직, 데이터 환경은 어느 정도 있지만 전담 인력의 부재
### 배경
* 글로벌 회사이고 전사적으로 데이터 레이크와 분석 플랫품은 구축되어 있습니다.
* 예를 들면, In-house로 하둡/Spark 클러스터, jupyter와 유사한 분석 노트북&리포팅 플랫폼, GA와 동등한 인하우스 클라이언트 로깅 플랫폼이 갖춰져 있었고 데이터도 이미 막대한 양이 적재되고 있습니다. 
* 문제는, 제품별로 데이터를 다루는 전담인력이 없던 상황이었고 데이터를 제대로 활용하지 못하고 있습니다.
* 이 조직에서 당면한 문제는 이미 도구와 데이터가 어느 정도 준비된 상황에서 데이터를 잘 활용하는 것입니다.

### 팀 빌딩 전략
* 지표 설계와 적재를 할 수 있는 데이터 분석가(DA)와 Analytics Engineering[^ae] 역할을 할 수 있는 엔지니어가 필요합니다.
* 처음부터 의미있는 인사이트를 도출하는 것을 목표로 하기 보다는 조직의 현황을 지표로 가시화할 수 있도록 하고 조직 내 이해관계자에게 이를 리포팅하는 것을 목표로 합니다.
* 조직의 여러 이해관계자로부터 지표현황에 대한 문의가 많아지고 신뢰가 쌓아가는 것을 다음 목표로 합니다.
* 이 후에는 조직 내 데이터팀에 대한 신뢰가 쌓이면 서비스 성장을 위한 여러가지 접근을 할 수 있습니다.

## 사례2: 작은 규모 조직, 준비된 것이 아무것도 없는 상황
### 배경
* 데이터 인프라가 거의 구축되어 있지 않거나 읽기전용 서비스 DB에 직접 접근합니다.
* 데이터 전담 인력이 없어 개발팀에서 admin으로 기획자들이 요청하는 기본지표를 제공하는 정도입니다.
* 서비스 기획을 데이터 기반에서 하고 있지 못합니다.

### 팀 빌딩 전략
* 신규로 데이터 엔지니어를 채용 가능하다면 채용하거나, 기존 백엔드 개발팀에서 리소스를 요청하여 지원을 받습니다.
* 가능하다면 지표 설계 경험이 있는 분석가를 채용하거나 어렵다면 기존에 기획업무를 담당하는 인력 중 관심자가 지표 설계 업무를 진행합니다.
* 초기에는 ad-hoc하게 진행하던 데이터 추출 업무 등을 자동화하며 기본적인 데이터 분석 인프라 구축을 하는데 노력을 기울입니다.
* 분석 담당자는 기존에 조직 내에서 활용중인 데이터를 정리하고 지표를 설계합니다.

[^ae]: https://www.getdbt.com/what-is-analytics-engineering/

**Reply by jongikp** [reply]
와.... 👏
